apps/web/src/services/dashboards/DashboardsLNB.vue

```diff
@@ -140,10 +146,10 @@ const filterFavoriteItems = (menuItems: LNBMenu[] = []): LNBMenu[] => {
     const result = [] as LNBMenu[];
     menuItems.forEach((d) => {
         if (Array.isArray(d)) {
-            const filtered = d.filter((menu) => (menu.id && state.favoriteItemMap[menu.id]) || menu.type !== MENU_ITEM_TYPE.ITEM);
+            const filtered = d.filter((menu) => (menu.id && state.favoriteItemMap[menu.favoriteOptions?.id || menu.id]) || menu.type !== MENU_ITEM_TYPE.ITEM);
             const hasProject = filtered.filter((f) => f.type === 'item').length > 0;
             if (hasProject) result.push(filtered);
-        } else if ((d.id && state.favoriteItemMap[d.id]) || d.type !== MENU_ITEM_TYPE.ITEM) result.push(d);
+        } else if ((d.id && state.favoriteItemMap[d.favoriteOptions?.id || d.id]) || d.type !== MENU_ITEM_TYPE.ITEM) result.push(d);
```

Comment:
question: why check both `d.id `and `favoriteOptions?.id` from `state.favoriteItemMap`?
I think just checking `favoriteOptions.id` is okay because they are set to workspace and project dashboard items.
Did I miss something? 😅